tests/config/test_templated_config.py

```diff
@@ -468,3 +468,16 @@ def test_simple_replace(self, val, format_dict, expected):
     def test_raises_error(self, val, format_dict, expected_error_message):
         with pytest.raises(ValueError, match=expected_error_message):
             _format_object(val, format_dict)
+
+    def test_customised_patterns(self, tmp_path, template_config):
+        config_loader = TemplatedConfigLoader(
+            str(tmp_path),
+            globals_dict=template_config,
```

Comment:
```suggestion
```
Should we remove the `globals_dict` argument? It doesn't seem to relate to this particular test.

Reply:
Good spot, I blindly copy-pasted 🤦‍♀️